index.html

```diff
@@ -73,88 +74,65 @@ <h1 class="recent">My Recent Works</h1>
-                <li><button class="lang-btns">JavaScript</button></li>
-                <li><button class="lang-btns">html</button></li>
+            <ul class="languages customlng2">
                 </ul>
         </section>
-        <section class="flexible-items-inside-grid"><button class="prjct-btn">See Project</button></section>
+        <section class="flexible-items-inside-grid"><button onclick="modalCreation(1)" class="prjct-btn">See Project</button></section>
     </div>
 
     <div class="articles">
-        <section class="flexible-items-inside-grid"><img class="project-logo" src="./cards/prjct_logo.png" alt="project-temp-logo"></section>
-        <section class="flexible-items-inside-grid"><h2 class="project-name">Multi-Post Stories<br>Gain+Glory</h2></section>
+        <section class="flexible-items-inside-grid"><img class="project-logo" src="./" alt="project-temp-logo"></section>
+        <section class="flexible-items-inside-grid"><h2 class="project-name"></h2></section>
         <section class="flexible-items-inside-grid">
-            <ul class="languages">
-                <li><button class="lang-btns">Ruby on rails</button></li>
-                <li><button class="lang-btns">css</button></li>
-                <li><button class="lang-btns">JavaScript</button></li>
-                <li><button class="lang-btns">html</button></li>
+            <ul class="languages customlng3">
                 </ul>
         </section>
-        <section class="flexible-items-inside-grid"><button class="prjct-btn">See Project</button></section>
+        <section class="flexible-items-inside-grid"><button onclick="modalCreation(2)" class="prjct-btn" >See Project</button></section>
     </div>
 
     <div class="articles">
-        <section class="flexible-items-inside-grid"><img class="project-logo" src="./cards/prjct_logo.png" alt="project-temp-logo"></section>
-        <section class="flexible-items-inside-grid"><h2 class="project-name">Multi-Post Stories<br>Gain+Glory</h2></section>
+        <section class="flexible-items-inside-grid"><img class="project-logo" src="./" alt="project-temp-logo"></section>
+        <section class="flexible-items-inside-grid"><h2 class="project-name"></h2></section>
         <section class="flexible-items-inside-grid">
-            <ul class="languages">
-                <li><button class="lang-btns">Ruby on rails</button></li>
-                <li><button class="lang-btns">css</button></li>
-                <li><button class="lang-btns">JavaScript</button></li>
-                <li><button class="lang-btns">html</button></li>
+            <ul class="languages customlng4">
                 </ul>
         </section>
-        <section class="flexible-items-inside-grid"><button class="prjct-btn">See Project</button></section>
+        <section class="flexible-items-inside-grid"><button onclick="modalCreation(3)" class="prjct-btn">See Project</button></section>
```

Comment:
- [ ] Please use the dom selector query to add dom onclick events.

Reply:
- [ ] Kindly make the change on all `See Project` buttons.

Reply:
Done, Thank you for your code review :+1: